src/app/component/shared/detail-view/detail-view.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from "@angular/core";
+import { FormlyFieldConfig } from "@ngx-formly/core";
+import { AbstractModel } from "src/app/models/AbstractModel";
+
+@Component({
+  selector: "app-detail-view",
+  template: `
+    <div>
+      <div *ngFor="let field of fields">
+        <div class="row">
+          <div class="col-sm-3">
+            <p class="text-left text-sm-right">
+              <strong>
+                {{ field.templateOptions.label }}
+              </strong>
+            </p>
+          </div>
+          <div class="col-sm-9">
+            <app-view [view]="model[field.key]"></app-view>
```

Comment:
`app-view` seems like a very general name. Are we viewing an entire application?